packages/wrangler/src/pages/functions/buildWorker.ts

```diff
@@ -367,6 +372,39 @@ export function buildNotifierPlugin(onEnd: () => void): Plugin {
 	};
 }
 
+/**
+ *
```

Comment:
Is this missing a comment?

Reply:
oh yeah...I was meaning to explain why we needed this plugin to begin with. Thank you for catching that!